manifests/3.0.0/opensearch-dashboards-3.0.0-test.yml

```diff
@@ -0,0 +1,52 @@
+---
+schema-version: '1.0'
+name: OpenSearch Dashboards
+ci:
+  image:
+    name: opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2
+components:
+  - name: OpenSearch-Dashboards
+    bwc-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: notificationsDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: observabilityDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: reportsDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: ganttChartDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: queryWorkbenchDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: securityDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
```

Comment:
I dont think `securityDashboards` will have tests without-security. Can you confirm

Reply:
@kavilla @tianleh Can you guys take a look and suggest changes to this?
Thanks!

Reply:
Tagging security dashboards maintainers: @cwperks @peternied @davidlago

Reply:
I'm not sure how these tests are being sourced, this file looks different from 2.7.0 version, https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.7.0/opensearch-dashboards-2.7.0-test.yml , has there been changes to the architecture?

There are security tests in the `functionalTestDashboards` repo, which allows for running with and without security enabled.

Reply:
Yes the architecture is changing. Thanks for asking this. Looks like we need to add documentation for new architecture! @Divyaasm Can you take this question up and also later take care of adding the documentation.

Reply:
Yes @peternied , we have recently updated the implemention for OSD integtests, this used run in a sequential order for all the OSD component plugins listed in ftrepo based on functionalTestDashboards input earlier. Now OSD integtest works similar to OS intgtests. Refer the related PR's https://github.com/opensearch-project/opensearch-build/pull/3316 and https://github.com/opensearch-project/opensearch-build/pull/3465 to get an idea on the implementation changes.

We will now be able to to run this individually for each OSD component or all together at once in parallel from Jenkins perspective and update results individually for each component in the S3 bucket based on the updated test manifest similar to the current PR. Let me know if you have any questions.

This PR is just used to test the latest implementation from Prod Jenkins.

Thanks,
Divya Madala

Reply:
OK, well please go ahead and include the tests as has been done with other plugins.  I am not certain this is correct, but I think its best to move forward with generation results than waiting on making a determination

Reply:
Thanks for the input!